### ISSUE TITLE
cleanup fails when kubectl is not present

### ISSUE THREAD

**Issue by vpavlin** [fabric8io/gofabric8#351]
I am using `oc cluster up` to run my openshift which means I don't have `kubectl` command in the VM.

`gofabric8 cleanup` fails with

```
failed to delete Deployments: exec: "kubectl": executable file not found in $PATH
```

Could you add check for `kubectl` existance in https://github.com/fabric8io/gofabric8/blob/master/cmds/cleanup.go#L224 and use `oc` if it is not available.

Not sure if it's worth the work as that function is just workaround for https://github.com/fabric8io/gofabric8/issues/248, but if the workaround sticks around longer, it would be nice.

**Comment by rawlingsj** [comment]
Yeah there is an assumption that gofabric8 was used to install and setup minishift / minikube.  We can catch the error and just download kubectl instead by programatically running the `gofabric8 install` instead.

**Comment by vpavlin** [comment]
My workaround is `ln -s /usr/bin/oc /usr/bin/kubectl`, but that's pretty dirty:)
